frontend/src/global_styles/openproject/_mixins.sass

```diff
@@ -214,3 +214,11 @@ $scrollbar-size: 10px
       cursor: default
       border-color: #c7c7c7
       background: rgba(218,223,225,0.75)
+
+@mixin unset-button-styles
```

Comment:
In the future we could think about having one mixin per file, like we have one BEM block per file